.github/workflows/installers-conda.yml

```diff
@@ -262,33 +283,48 @@ jobs:
         run: |
           $PKG_PATH -b
 
-          root_prefix=$(compgen -G $HOME/.local/spyder-*)
+          base_prefix=$(compgen -G $HOME/.local/spyder-*)
+          spy_rt=$base_prefix/envs/spyder-runtime
+          menu=$spy_rt/Menu/spyder-menu.json
+          mode=user
+          shortcut=$(${base_prefix}/bin/python - <<EOF
+          from menuinst.api import _load
+          menu, menu_items = _load("$menu", target_prefix="$spy_rt", base_prefix="$base_prefix", _mode="$mode")
+          print(menu_items[0]._paths()[0])
+          EOF
+          )
 
           echo "Install info:"
-          echo "Contents of ${root_prefix}:"
-          ls -al $root_prefix
-          echo -e "\nContents of ${root_prefix}/uninstall-spyder.sh:"
-          cat ${root_prefix}/uninstall-spyder.sh
-
-          shortcut_path=/share/applications/spyder_spyder.desktop
-          [[ -e ${root_prefix}/.nonadmin ]] && shortcut_path=${HOME}/.local${shortcut_path} || shortcut_path=/usr${shortcut_path}
-
-          if [[ -e $shortcut_path ]]; then
-              echo -e "\nContents of ${shortcut_path}:"
-              cat $shortcut_path
+          echo "Contents of ${base_prefix}:"
+          ls -al $base_prefix
+          echo -e "\nContents of ${base_prefix}/uninstall-spyder.sh:"
+          cat ${base_prefix}/uninstall-spyder.sh
+
+          if [[ -e $shortcut ]]; then
+              echo -e "\nContents of ${shortcut}:"
+              cat $shortcut
           else
-              echo "$shortcut_path does not exist"
+              echo "$shortcut does not exist"
               exit 1
           fi
 
       - name: Test Windows Installer
         if: runner.os == 'Windows'
         shell: cmd
         run: |
+          set base_prefix=%USERPROFILE%\AppData\Local\spyder-6
           start /wait %PKG_PATH% /InstallationType=JustMe /NoRegistry=1 /S
-
-          set "shortcut_path=%USERPROFILE%\AppData\Roaming\Microsoft\Windows\Start Menu\Programs\spyder\Spyder.lnk"
-          if exist "%shortcut_path%" (
+          if exist %base_prefix%\install.log type %base_prefix%\install.log
+
+          set spy_rt=%base_prefix%\envs\spyder-runtime
+          set menu=%spy_rt%\Menu\spyder-menu.json
+          set mode=system
+          for /F "tokens=*" %%i in (
+              '%base_prefix%\python -c "from menuinst.api import _load; menu, menu_items = _load(r'%menu%', target_prefix=r'%spy_rt%', base_prefix=r'%base_prefix%', _mode='%mode%'); print(menu_items[0]._paths()[0])"'
+          ) do (
+              set shortcut=%%~fi
+          )
```

Comment:
I think it should be possible to use the same script I mentioned above to replace this blob of batch code, right?

Reply:
Yes, I think you are correct.